public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -0,0 +1,466 @@
+            this.getSurvey(); });
+    }
+  }
+
+  handleChange(val) {
+        this.setState({ searchWord: val });
+        this.searchCount(val);
+        this.search(val);
+  }
+
+  getSurvey() {
+        const {survey, searchWord, extraQuestionConceptIds} = this.state;
+        api.getSurveyQuestions(survey.conceptId.toString(), searchWord, extraQuestionConceptIds).then(
+            (x: any) => {
+                this.processSurveyQuestions(x);
+        });
+  }
+
+  processSurveyQuestions(results: any) {
+        const survey = results.survey;
+        const questions = results.questions.items;
+        this.setDefaults(questions, 0, survey);
+  }
+
+  setDefaults(surveyQuestions: any, level: any, survey) {
+        const {isCopeSurvey} = this.state;
+        for (const q of surveyQuestions) {
+            q.actualQuestionNumber = q.questionOrderNumber;
+            if (isCopeSurvey) {
+                q.graphToShow = GraphType.SurveyVersion;
+                q.selectedAnalysis = q.versionAnalysis;
+            } else {
+                q.graphToShow = GraphType.BiologicalSex;
+                q.selectedAnalysis = q.genderAnalysis;
+            }
+            q.graphDataToShow = 'Count';
+            q.resultFetchComplete = false;
+        }
+        surveyQuestions.sort((a1, a2) => {
+            if (a1.actualQuestionNumber < a2.actualQuestionNumber) {
+                return -1;
+            }
+            if (a1.actualQuestionNumber > a2.actualQuestionNumber) {
+                return 1;
+            }
+            return 0;
+        });
+        this.setState({questions: surveyQuestions, survey: survey, loading: false});
+  }
+
+  backToMain() {
+        navigateByUrl('');
+  }
+
+
+  render() {
+        const {loading, searchWord, isCopeSurvey, survey, questions, surveyVersions, surveyPdfUrl, showStatement} = this.state;
+        const statClass = isCopeSurvey ? 'cope-stat-layout' : 'stat-layout';
+        const statStyle = isCopeSurvey ? styles.copeStatLayout : styles.statLayout;
+        return <React.Fragment>
+        <style>{surveyStyle}</style>
+        <div className='survey-view' style={styles.surveyView}>
+        {survey && <SurveyDescReactComponent surveyName={survey.name} isCopeSurvey={isCopeSurvey} surveyDescription={survey.description}
+        click={this.showCopeStatement}/>
+        }
+        <div className='search-bar-container' style={styles.searchBarContainer}>
+            <SearchComponent value={searchWord || ''} searchTitle=''
+                onChange={(val) => this.handleChange(val)}
+                onClear={() => this.handleChange('')} />
+        </div>
+        {loading && <Spinner />}
+        {survey &&
+        <section className='results' style={styles.results}>
+        <a className='btn btn-link btn-sm main-search-link' style={styles.searchLink} onClick={() => this.backToMain()}>
+        {'< '} Back to main search </a>
```

Comment:
Nit: can use the html entity code here:
```suggestion
        &lt; Back to main search </a>
```

Reply:
Changed this thanks.